mod/match.php

```diff
@@ -123,9 +96,37 @@ function match_content(App $a)
 	$tpl = Renderer::getMarkupTemplate('viewcontact_template.tpl');
 	$o = Renderer::replaceMacros($tpl, [
 		'$title'    => DI::l10n()->t('Profile Match'),
-		'$contacts' => $entries,
-		'$paginate' => $paginate
```

Comment:
Please remove the unused template variable in the template file as well.

Reply:
The template is used in several places, some with, some without pagination.

Reply:
Got it.